autoload/slime.vim

```diff
@@ -351,13 +351,17 @@ endfunction
 function! slime#send_cell(cell_delimiter) abort
   let line_ini = search(a:cell_delimiter, 'bcnW')
   let line_end = search(a:cell_delimiter, 'nW')
+  let line_ini_delta = 1
+  let line_end_delta = 1
   if !line_ini
       let line_ini = 1
+      let line_ini_delta = 0
   endif
   if !line_end
       let line_end = line("$")
+      let line_end_delta = 0
   endif
-  call slime#send_range(line_ini, line_end)
+  call slime#send_range(line_ini+line_ini_delta, line_end-line_end_delta)
```

Comment:
how's this?

```vim
  " line after delimiter or top of file
  let line_ini = line_ini ? line_ini + 1 : 1
  " line before delimiter or bottom of file
  let line_end = line_end ? line_end - 1 : line("$")
  call slime#send_range(line_ini, line_end)
```

Reply:
perfect !

Reply:
tested locally, works for me … not sure I covered all the use-cases

- between 2 delimiters
- top of file and delimiter
- delimiter and bottom of file
- no delimiters (whole file)

@mdedonno1337 can you confirm?

Reply:
Yes.

Added the check if the cell is empty (two separators, no content) or not.